recipes/bioconductor-mzr/meta.yaml

```diff
@@ -20,8 +20,8 @@ build:
 # SystemRequirements: C++11, GNU make
 requirements:
   host:
-    - 'bioconductor-biobase >=2.44.0,<2.45.0'
-    - 'bioconductor-biocgenerics >=0.30.0,<0.31.0'
+    - 'bioconductor-biobase'
+    - 'bioconductor-biocgenerics'
```

Comment:
This is done to comply with how bioconductor pinning should be done (also at the request of the bioconductor core team), please do not undo it. I'll update your PR shortly to add the 3.8 version in.